src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu-item/responsive-vertical-menu-item.style.ts

```diff
@@ -160,6 +160,18 @@ export const StyledResponsiveMenuItem = styled.button<StyledResponsiveMenuItemPr
       color: var(--colorsUtilityYang080);
       cursor: default;
     `}
+
+
+  ${({ depth }) =>
+    depth >= 2 &&
+    css`
+      color: var(--colorsUtilityYang080);
+      font-weight: var(--fontWeights400) !important;
```

Comment:
comment: I think it'd be better use `&& {...}` to raise the specificity as using `!important` could create weird issues